### PR TITLE
OLS: added bundle build for lightspeed operator

### PR DESCRIPTION
**Pull request by xrajesh**
> Add bundle image for test
> Publish bundle image with latest tag

### CODE REVIEW

**Comment by bparees** [comment]
/pj-rehearse pull-ci-openshift-lightspeed-operator-main-bundle-e2e

File: ci-operator/config/openshift/lightspeed-operator/openshift-lightspeed-operator-main.yaml [hunk 1]
```diff
@@ -3,6 +3,18 @@ base_images:
     name: golangci-lint
     namespace: ci
     tag: v1.54.2
+  lightspeed-console-plugin:
+    name: lightspeed-console-plugin
+    namespace: openshift
```
Comment (bparees) [inline ci-operator/config/openshift/lightspeed-operator/openshift-lightspeed-operator-main.yaml:8]: i think these need to be "ols", as that is where we are promoting to, not openshift

**Comment by xrajesh** [comment]
/pj-rehearse pull-ci-openshift-lightspeed-operator-main-bundle-e2e

**Comment by xrajesh** [comment]
/pj-rehearse pull-ci-openshift-lightspeed-operator-main-bundle-e2e

**Comment by xrajesh** [comment]
/pj-rehearse pull-ci-openshift-lightspeed-operator-main-bundle-e2e

**Comment by bparees** [comment]
i don't know why this is not being found:

``
$ oc get is/lightspeed-service-api -n ols
NAME                     IMAGE REPOSITORY                                       TAGS     UPDATED
lightspeed-service-api   registry.ci.openshift.org/ols/lightspeed-service-api   latest   6
```

you may need to talk to the test platform team to get some help

**Comment by xrajesh** [comment]
> i don't know why this is not being found:
> 
> ```
> $ oc get is/lightspeed-service-api -n ols
> NAME                     IMAGE REPOSITORY                                       TAGS     UPDATED
> lightspeed-service-api   registry.ci.openshift.org/ols/lightspeed-service-api   latest   6
> ```
> 
> you may need to talk to the test platform team to get some help

@bparees where did you run this command - How can I check this ?

**Comment by bparees** [comment]
I logged into the app.ci cluster which runs registry.ci.openshift.org:

https://docs.ci.openshift.org/docs/how-tos/use-registries-in-build-farm/

**Comment by xrajesh** [comment]
/pj-rehearse pull-ci-openshift-lightspeed-operator-main-bundle-e2e

**Comment by xrajesh** [comment]
/pj-rehearse pull-ci-openshift-lightspeed-operator-main-bundle-e2e

**Comment by xrajesh** [comment]
/pj-rehearse pull-ci-openshift-lightspeed-operator-main-bundle-e2e

**Comment by xrajesh** [comment]
/pj-rehearse pull-ci-openshift-lightspeed-operator-main-bundle-e2e

**Pull request merged**